README.md

```diff
@@ -420,7 +422,8 @@ rule with lower `priority` will be matched first;
 * `THEFUCK_HISTORY_LIMIT` &ndash; how many history commands will be scanned, like `2000`;
 * `THEFUCK_ALTER_HISTORY` &ndash; push fixed command to history `true/false`;
 * `THEFUCK_WAIT_SLOW_COMMAND` &ndash; max amount of time in seconds for getting previous command output if it in `slow_commands` list;
-* `THEFUCK_SLOW_COMMANDS` &ndash; list of slow commands, like `lein:gradle`.
+* `THEFUCK_SLOW_COMMANDS` &ndash; list of slow commands, like `lein:gradle`;
+* `THEFUCK_NUM_CLOSE_MATCHES` &ndash; maximum number of close matches to suggest, like `5`.
```

Comment:
Should we just switch to putting a `;` on the last line (instead of `.`), so the diff can be smaller going forward?

Reply:
I had the exact same thought. But the next thought was "Wait. It's text, not code". Perhaps it makes sense to have a period there. For me it will look like an unfinished list.

Reply:
I believe that's a good one, reasonable change. :+1:

Reply:
Thanks, I'll wait until this is closed to make that change (if I remember)

Reply:
@josephfrazier I think it should be ok to remove `;`, as it's not required by grammar rules